packages/scaffolding-cli/templates/build/azDevops/azure/azure-pipelines-ssr-aks.yml

```diff
@@ -206,25 +205,30 @@ stages:
               terraform_extra_properties: {
                 TF_VAR_resource_group_location: uksouth,
                 TF_VAR_use_existing_resource_group: true,
-                TF_VAR_resource_group_name: $(conventional_resource_namer),
+                TF_VAR_resource_group_name: amido-stacks-nonprod-uks-node,
                 TF_VAR_name_company: $(company),
                 TF_VAR_name_project: $(project),
                 TF_VAR_name_component: $(component),
                 TF_VAR_create_cosmosdb: false,
+                TF_VAR_create_cache: true,
                 TF_VAR_name_environment: dev,
                 TF_VAR_name_stage: dev,
-                TF_VAR_app_gateway_frontend_ip_name: $(conventional_resource_namer),
+                TF_VAR_app_gateway_frontend_ip_name: amido-stacks-nonprod-uks-node,
```

Comment:
Hardcoded?

Reply:
yeah we replace it wholesale now with MACRO_TOKEN so user will need to replace themselves